docs/android-app-bundle/build-apks.md

```diff
@@ -27,15 +27,15 @@ Path to the keystore to sign the apk files with
 ##### `--ks-pass=KEYSTORE_PASSWORD`
 
 
-Keystore password. If not given, the value will be checked from environment variable `KEYSTORE_PASSWORD`. Alternatively to entering` KEYSTORE_PASSWORD `in plaintext, it may also be specified using a `@env:` prefix followed by a environment variable name, or `@file:` prefix followed by a path to the file containing the value. Example: `@env:<variable>` uses the value in the environment variable named `<variable>`, and `@file:<file_path>` uses the value from file at `<file_path>`. [Default: None]
+Keystore password. If not given, the value will be checked from the environment variable `KEYSTORE_PASSWORD`. Alternatively to entering `KEYSTORE_PASSWORD` in plaintext, it may also be specified using the `@env:` prefix followed by an environment variable name, or the `@file:` prefix followed by a path to the file containing the value. Example: `@env:<variable>` uses the value in the environment variable named `<variable>`, and `@file:<file_path>` uses the value from file at `<file_path>`. [Default: None]
```

Comment:
missing "the":
from **the** file at `<file_path>`

Reply:
Commited the missing "the" to the respective file

Reply:
Regenerated docs in a6a823a.